libkbfs/root_metadata_test.go

```diff
@@ -386,10 +410,6 @@ func makeFakeWriterMetadataFuture(t *testing.T) writerMetadataFuture {
 	}
 }
 
-func TestWriterMetadataUnknownFields(t *testing.T) {
```

Comment:
Oops, added back. I don't remember doing this, but I used keyboard macros to do some work and it might have messed up and removed this one.